Content.Server/Disease/DiseaseSystem.cs

```diff
@@ -186,6 +186,14 @@ private void CureDisease(DiseaseCarrierComponent carrier, DiseasePrototype disea
             _popupSystem.PopupEntity(Loc.GetString("disease-cured"), carrier.Owner, Filter.Entities(carrier.Owner));
         }
 
+        public void CureAllDiseases(DiseaseCarrierComponent carrier)
```

Comment:
Should follow the [conventions for public APIs in Entity Systems](https://docs.spacestation14.io/en/getting-started/conventions#public-api-method-signature).